site/content/contribute/server/developer-setup/windows-wsl.md

```diff
@@ -85,7 +100,9 @@ This is an unofficial guide. Community testing, feedback and improvements are we
     ```
 
 ### Troubleshooting:
-1. If you see an error like `the input device is not a TTY.  If you are using mintty, try prefixing the command with 'winpty'`.  Reinstall git for windows and make sure you choose `Use Windows' default console window` instead of `Use MinTTY`
+
+1. If you see an error like `the input device is not a TTY.  If you are using mintty, try prefixing the command with 'winpty'`.  Reinstall git for windows and make sure you choose `Use Windows' default console window` instead of `Use MinTTY`.
```

Comment:
```suggestion
1. If you see an error like `the input device is not a TTY.  If you are using mintty, try prefixing the command with 'winpty'`, then reinstall Git for Windows and make sure you choose **Use Windows' default console window** instead of **Use MinTTY**.
```